styles.css

```diff
@@ -1022,3 +1027,610 @@
     display: none;
   }
 }
+
+@media only screen and (min-width: 768px) {
+  body {
+    background-color: #bbb8b5;
+    margin: 0%;
+    width: 100%;
+  }
+
+  .appbar-compact {
+    display: flex;
+    width: 100%;
+    background-color: #fff;
+  }
+
+  .toolbar {
+    display: flex;
+    gap: 56%;
+    width: 100%;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .nav-list {
+    display: flex;
+    flex-direction: row;
+    gap: 12px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    color: #344563;
+    padding: 0;
+    width: 13%;
+  }
```

Comment:
- [ ] Good job adding a proper navlist according to the Figma design :clap: , however I noticed that when you are clicking on the navlinks you are not redirected to a specific section, kindly make sure that when the user click on the navlink is being redirected to the specific class. For more information please click [here] (https://www.geeksforgeeks.org/how-to-redirect-to-a-particular-section-of-a-page-using-html-or-jquery/) :smile:

![Screenshot from 2023-03-02 10-55-06](https://user-images.githubusercontent.com/80458199/222382298-b4a1658f-99ea-4795-bfd3-ba3dc541c3e2.png)